File: python/MustafaMuratCoskun/Work/Döngüler/atm.py
print("""
**********************
ATM'ye Hoşgeldiniz.
**********************
""")



print("""
Lütfen Yapmak İstediğiniz İşlem Numarasını Giriniz.
---------------------------------------------------
1) Para Çekme
2) Para Yatırma
3) Para Gönderme
4) Hesap Bilgileri

Çıkmak İçin 'Q' ya basınız veya 'Çıkış' tuşuna basınız.

""")

hesapNumarasi = 1345233
hesapSahibiAdi = "Ömer Faruk"
hesapSahibiSoyadi = "Özmen"
telefon = "05414048256"
bakiye = 1000

while True:
    islem = input("İşlemi Seçiniz.")
    if (islem== "q" and "Çıkış"):
        print("Yine Bekleriz...")
        break
    elif (islem == "1"):
        paraCekme = int(input("Çekmek İstediğiniz Tutarı Giriniz."))
        bakiye = bakiye - paraCekme
        print("{} TL Çekildi. Kalan Bakiye {}".format(paraCekme,bakiye))
    elif(islem == "2"):
        paraYatirma = int(input("Yatırmak İstediğiniz Tutarı Giriniz."))
        bakiye = bakiye + paraYatirma
    elif (islem == "3"):
        print("""Parayı Nasıl Göndermek İstediğinizi Seçiniz.""")
        print("""
        ***********************
        1) Hesap Numarası
        2) İBAN
        3) T.C numarası
        4) Geri
        ***********************
        """)
        paraislem = input("Seçim Yapınız.")
        if (paraislem == "1"):
            paraGondermeHesap = int(input("Hesap Numarasını Giriniz."))
            hesabaGidenPara = int(input("Göndermek İstediğiniz Tutarı Giriniz."))
            bakiye = bakiye - hesabaGidenPara
            print("{} Hesap Numarasına {} TL gönderildi.\nBir Üst Menüye Yönlendiriliyorsunuz.".format(paraGondermeHesap,hesabaGidenPara))
        elif (paraislem == "2"):
            ibanNumarasi = int(input("İban Numarasını Giriniz."))
            if ((ibanNumarasi) != 16):
                print("İban Numarası 16 Haneli Olmalıdır.")
            ibanaGidenPara = int(input("Göndermek İstediğiniz Tutarı Giriniz."))
            bakiye = bakiye - ibanaGidenPara
            print("{} İBAN Numarasına {} Tl gönderildi.\nBir Üst Menüye Yönlendiriliyorsunuz.".format(ibanNumarasi,ibanaGidenPara))
        elif (paraislem == "3"):
            tcNumarasi = int(input("T.C. Kimlik Numarası giriniz."))
            if (len(tcNumarasi) != 11):
                print("TC no 11 Haneli Olmalıdır.")
            tcGidenPara = int(input("Göndermek İstediğiniz Tutarı Giriniz."))
            bakiye = bakiye - tcGidenPara
            print("{} T.C. Kimlik Numarasına {} Tl Gönderilmiştir.\nBir Üst Menüye Yönlendiriliyorsunuz.".format(tcNumarasi,tcGidenPara))
        elif (paraislem == "4"):
            print("Ana Menüye Yönlendiriliyorsunuz.")
            break
            
        
    elif (islem == "4"):
        print("""
        ******************
        Hesap Bilgileriniz
        ******************
        Hesap Numarası: {}
        Ad Soyad: {} {}
        Telefon: {}
        Hesap Bakiyesi: {}
        """.format(hesapNumarasi,hesapSahibiAdi,hesapSahibiSoyadi,telefon,bakiye))
        break

            

File: java/MustafaMuratCoskun/OOPAtm/src/ATM.java
import java.util.Scanner;

public class ATM {
	public void calis(Hesap hesap) {
		Login login = new Login();

		Scanner scanner = new Scanner(System.in);

		System.out.println("Bankamıza Hoşgeldiniz...");
		System.out.println("Kullanıcı Girişi");

		int girisHakki = 3;

		while (true) {
			if (login.login(hesap)) {
				System.out.println("Giriş Başarılı...");
				break;
			} else {
				System.out.println("Giriş Başarısız.");
				girisHakki -= 1;
				System.out.println("Kalan giriş hakkı " + girisHakki);
			}
			if (girisHakki == 0) {
				System.out.println("Giriş Hakkınız Bitti");
				return;
			}
		}
		String islemler = "1) Bakiye Görüntüle\n" + "2) Para Yatırma\n" + "3) Para Çekme\n" + "Çıkış için q";
		System.out.println(islemler);

		while (true) {
			System.out.print("İşlemi Seçiniz");
			String islem = scanner.nextLine();

			if (islem.equals("q")) {
				System.out.println("Çıkış Yapılıyor...");
				break;
			} else if (islem.equals("1")) {
				System.out.println("Merhaba " + hesap.getKullaniciAdi() + " Bakiyeniz " + hesap.getBakiye());
			} else if (islem.equals("2")) {
				System.out.print("Yatırmak İstediğiniz Tutar");
				int tutar = scanner.nextInt();
				scanner.nextLine();
				hesap.paraYatir(tutar);
			} else if (islem.equals("3")) {
				System.out.print("Çekmek istediğiniz Tutar");
				int tutar = scanner.nextInt();
				scanner.nextLine();
				hesap.paraCek(tutar);
			} else {
				System.out.println("Geçersiz İşlem Lütfen Doğru Bir Tuşlama Yapınız.");
			}
		}
	}

}
